community/modules/network/private-service-access/versions.tf

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 3.83"
+    }
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = ">= 3.83"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.0"
+    }
+  }
+  provider_meta "google" {
+    module_name = "blueprints/terraform/hpc-toolkit:private-service-access/v1.30.0"
+  }
+
+  provider_meta "google-beta" {
+    module_name = "blueprints/terraform/hpc-toolkit:private-service-access/v1.30.0"
```

Comment:
Please update to v1.31.0 or v1.31.1 depending on timing of https://github.com/GoogleCloudPlatform/hpc-toolkit/pull/2420